src/lib.rs

```diff
@@ -119,6 +128,7 @@ pub struct ClientBuilder<T: Runtime> {
     url: Option<String>,
     client: Option<jsonrpsee::Client>,
     page_size: Option<u32>,
+    mortal_period: Option<Option<u64>>,
```

Comment:
What's up with `Option<Option<_>>` rather than `Option<_>`?

Reply:
`None` => User has not set anything
`Some(64)` => User set a mortal period of 64
`Some(None)` => User has set it to immortal

Reply:
Ok, it would be more readable with an enum such as `MortalityPeriod`.